lib/hsa/hsaRfiMaskOutput.cpp

```diff
@@ -15,25 +15,11 @@ hsaRfiMaskOutput::hsaRfiMaskOutput(Config& config, const string& unique_name,
     _network_buf = host_buffers.get_buffer("network_buf");
     register_consumer(_network_buf, unique_name.c_str());
 
-    _lost_samples_buf = host_buffers.get_buffer("lost_samples_buf");
-    register_consumer(_lost_samples_buf, unique_name.c_str());
-
     _rfi_mask_output_buf = host_buffers.get_buffer("rfi_mask_output_buf");
     register_producer(_rfi_mask_output_buf, unique_name.c_str());
 
-    _output_buf = host_buffers.get_buffer("output_buf");
-    register_producer(_output_buf, unique_name.c_str());
-
-    // Config parameters
-    _num_local_freq = config.get<uint32_t>(unique_name, "num_local_freq");
```

Comment:
Read from config but never used?

Reply:
yeah that was part of the removed Pathfinder support that apparently didn't get cleaned up.